tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -668,7 +668,7 @@ describe(__filename, () => {
         location: fakeRouterLocation(),
       });
 
-      expect(extractId(ownProps)).toEqual(`foobar-1`);
+      expect(extractId(ownProps)).toEqual(`foobar-`);
```

Comment:
That's kind of weird. Should a null value default to 1 instead of blank? Or maybe the string should be something other than blank? `noPageSet` or something? I dunno, maybe it doesn't matter 🤷‍♂️

Reply:
I agree it looks weird, but there doesn't seem to be an actual reason to change it, as all we're doing is creating an `extractId` which nobody will ever see.

Reply:
Fair enough 😄